WORKSPACE

```diff
@@ -21,3 +21,8 @@ gazelle_dependencies()
 load("@io_bazel_rules_go//tests:bazel_tests.bzl", "test_environment")
 
 test_environment()
+
+load("//third_party:repos.bzl", "go_repositories")
```

Comment:
I'd rather not introduce a new macro for this. Repos that `gazelle`, `fetch_repo`, and other binaries depend on directly should be in `gazelle_dependencies`. This will make those repositories available for anyone that uses Gazelle, so the set should be pretty minimal, and we should keep using `_maybe`.

It's fine to declare test dependencies directly in WORKSPACE, since other repositories don't need these.

Reply:
Ahh makes sense, my bad for not catching this.

Looks like we there are no unique test dependencies, so we don't need anything new in WORKSPACE.